src/components/AssetsTable/index.jsx

```diff
@@ -19,12 +19,18 @@ export class AssetsTable extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      modalOpen: false,
       assetToDelete: {},
+      copyButtonIsClicked: false,
       deletedAsset: {},
       deletedAssetIndex: null,
       elementToFocusOnModalClose: {},
-      copyButtonIsClicked: false,
+      modalOpen: false,
+      statusAlertFields: {
+        alertType: 'info',
+        alertDialog: '',
```

Comment:
Nit: Should we alphabetize the fields of this object too?

Reply:
Yes :)